OracleIdentityGovernance/samples/containerizedDB/docker-compose.yaml

```diff
@@ -0,0 +1,102 @@
+#
+## Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+##
+## Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+##
+## Author: Kaushik C
+##
+## Description: script to build a Docker image for Oracle Access Manager 
+##
+##
+## DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+##
+##
+#
+version: "3.0"
+
+services:
+  #
+  # The Oracle DB Definition
+  #
+  oimdb:
+    image: ${DC_REGISTRY_DB}/oracle/database:12.2.0.1-ee
```

Comment:
Please use the `19.3.0.0` tag from Oracle Container Registry. The `12.2.0.1` build is too old.